CHANGES.rst

```diff
@@ -133,43 +146,56 @@ Other changes
   homeless builtins, to help with debugging, and in order to avoid a CPython
   warning according to https://bugs.python.org/issue20204
 
+.. _2056: https://github.com/cython/cython/issues/2056
+.. _3090: https://github.com/cython/cython/issues/3090
+.. _2227: https://github.com/cython/cython/issues/2227
+.. _3625: https://github.com/cython/cython/issues/3625
+.. _3664: https://github.com/cython/cython/issues/3664
+.. _3616: https://github.com/cython/cython/issues/3616
+.. _3693: https://github.com/cython/cython/issues/3693
+.. _3707: https://github.com/cython/cython/issues/3707
+.. _3514: https://github.com/cython/cython/issues/3514
+.. _3663: https://github.com/cython/cython/issues/3663
+.. _3754: https://github.com/cython/cython/issues/3754
+.. _3637: https://github.com/cython/cython/issues/3637
+.. _3736: https://github.com/cython/cython/issues/3736
```

Comment:
Why aren't these at the end of the file as well?

On a related note – I'm not sure if I really want to maintain this list whenever I add an entry. Including merge conflicts and all that. The linking seems like something that a Sphinx plugin should do for me.

Reply:
> The linking seems like something that a Sphinx plugin should do for me.

There is [sphix-issues](https://github.com/sloria/sphinx-issues), I will add it to the doc build. Then using `:issue:`3736` should automatically link to the issue.

Reply:
hmm, where *is* the doc build? I can't find a CI run that builds the docs. Does the readthe docs "advanced configuration" specify a `requirements.txt` ? (It should be at https://readthedocs.org/dashboard/cython/advanced/)?

Reply:
These are leftovers then, right?

> where is the doc build

It's probably set up somewhat manually on readthedocs. @robertwb originally created it.

Reply:
> These are leftovers then, right?

Removing